### PR TITLE
Contributing automation IncreaseIncidentSeverity.

### PR DESCRIPTION
**Pull request by tundisto**
## Status
- [ ] In Progress
- [x] Ready
- [ ] In Hold - (Reason for hold)

## Related Issues
N/A

## Description
Contributing a new automation 'IncreaseIncidentSeverity', which optionally increases an incident's severity to the new value, but only if greater than the existing severity.  This can save some extra steps in the playbook when one wants to only increase, but not decrease, incident severity, as they don't have to do a conditional check first.

## Screenshots
IncreaseIncidentSeverity-2
IncreaseIncidentSeverity-1

## Minimum version of Demisto
- [ ] 4.5.0
- [ ] 5.0.0
- [x] 5.5.0

## Does it break backward compatibility?
   - [ ] Yes
       - Further details:
   - [x] No

## Must have
- [x] Tests
- [x] Documentation 

## Demisto Partner?
N/A

### CODE REVIEW

File: Packs/CommonScripts/Scripts/IncreaseIncidentSeverity/IncreaseIncidentSeverity.yml [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+  required: true
+  auto: PREDEFINED
+  predefined:
+  - Unknown
+  - Informational
+  - Low
+  - Medium
+  - High
+  - Critical
+  - "0"
```
Comment (Shellyber) [inline Packs/CommonScripts/Scripts/IncreaseIncidentSeverity/IncreaseIncidentSeverity.yml:22]: No need to give an option to choose the severity numbers (0-4).
Please remove

File: Packs/CommonScripts/Scripts/IncreaseIncidentSeverity/IncreaseIncidentSeverity.js [hunk 2]
```diff
@@ -0,0 +1,25 @@
+const SEVERITY_VALUES = {
+  "unknown": 0,
+  "informational": 0.5,
+  "low": 1,
+  "medium": 2,
+  "high": 3,
+  "critical": 4,
+  "0": 0,
```
Comment (Shellyber) [inline Packs/CommonScripts/Scripts/IncreaseIncidentSeverity/IncreaseIncidentSeverity.js:8]: No need to format the severity numbers and use it in const.
Please remove.

**Comment by tundisto** [comment]
There is a reason — flexibility. There’s no need to limit the field to
either string or numeric value.  Both are equally valid. I shall keep them
both.

**Comment by Shellyber** [comment]
@tundisto The PR is missing Release Notes.
https://xsoar.pan.dev/docs/integrations/changelog

Also I can't run your code from Demisto.
Let's schedule a demo.

**Comment by tundisto** [comment]
When you say "can't run your code", what do you mean?

**Comment by Shellyber** [comment]
That the script doesn't work from Demisto

**Comment by tundisto** [comment]
Can you please provide more information than that?

**Comment by tundisto** [comment]
CircleCI says "The pack version needs to be raised".  Shall I bump it up or leave it as-is?

**Comment by Shellyber** [comment]
You can bump it

On Mon, 27 Jul 2020 at 20:21 Tim Underhay <notifications@github.com> wrote:

> CircleCI says "The pack version needs to be raised". Shall I bump it up or
> leave it as-is?
>
> —
> You are receiving this because you were mentioned.
> Reply to this email directly, view it on GitHub
> <https://urldefense.proofpoint.com/v2/url?u=https-3A__github.com_demisto_content_pull_8195-23issuecomment-2D664528803&d=DwMCaQ&c=V9IgWpI5PvzTw83UyHGVSoW3Uc1MFWe5J8PTfkrzVSo&r=qVFauV5Znio35RXTvoaxJAqZNAGn-RFNF0elNKU0Aow&m=z-iSLgsMCaz7puVnkeGYhpPulrP326Hx7hAQZquZz3o&s=xh9ZJokI37w_UKLPEUym9rZ-4Sg6nzhr30mKkZ7rSJU&e=>,
> or unsubscribe
> <https://urldefense.proofpoint.com/v2/url?u=https-3A__github.com_notifications_unsubscribe-2Dauth_AK6J23TV2S3FTPNV7ZZ6HVDR5WZRPANCNFSM4PGC62NQ&d=DwMCaQ&c=V9IgWpI5PvzTw83UyHGVSoW3Uc1MFWe5J8PTfkrzVSo&r=qVFauV5Znio35RXTvoaxJAqZNAGn-RFNF0elNKU0Aow&m=z-iSLgsMCaz7puVnkeGYhpPulrP326Hx7hAQZquZz3o&s=sLDB5uPB0ojZll0-RndVcWfN8AZJmJu0brmaFISeniA&e=>
> .
>
-- 

*Shelly Berman*

Software Engineer, Content Team, Demisto

Email: sberman@paloaltonetworks.com

**Pull request merged**